playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -132,6 +132,7 @@ EDXAPP_CAS_ATTRIBUTE_PACKAGE: ""
 EDXAPP_ENABLE_AUTO_AUTH: false
 # Settings for enabling and configuring third party authorization
 EDXAPP_ENABLE_THIRD_PARTY_AUTH: false
+EDXAPP_ENABLE_OAUTH2_PROVIDER: false
```

Comment:
The reason it needs to be off by default, is that this file is used for **all** ansible configs, not just the sandbox, so we don't want to cause new behaviour for people not interested in the sandbox config.

When we want to run a playbook for sandbox + ecommerce (not a common setup, it seems),  then we will set it to "true" in edx_sandbox_ecommerce_dogwood.yml.

Hope that helps?